Fix samples browser (#1631)
Azure-Samples/azure-search-openai-demo: samples/document-security/README.md

---
name: ChatGPT + Enterprise data + document security

> description: This guide demonstrates how to add an optional login and document level access control system to the sample. This system can be used to restrict access to indexed data to specific users.

Reviewer: ```suggestion
description: This guide demonstrates how to add an optional login and document level access control system to a RAG chat app. This system can be used to restrict access to indexed data to specific users.
```

languages:
- python
- typescript
- bicep
- azdeveloper
products:
- azure-openai
- azure-cognitive-search
- azure-app-service
- azure
page_type: sample
urlFragment: azure-search-openai-demo-document-security
---
# Setting up optional login and document level access control

> 

Reviewer: Possible first paragraph for this one:

"The [azure-search-openai-demo](https://github.com/Azure-Samples/azure-search-openai-demo/) project can set up a full RAG chat app on Azure AI Search and OpenAI so that you can chat on custom data, like internal enterprise data or domain-specific knowledge sets. For full instructions on setting up the project, consult the [main README](/README.md), and then return here for detailed instructions on configuring login and access control."

* It's possible that your tenant admin requires [admin approval of all new apps](https://learn.microsoft.com/azure/active-directory/manage-apps/manage-consent-requests). Regardless of whether you select the delegated or admin permissions, the app will not work without tenant admin consent.

## Adding data with document level access control

The sample supports 2 main strategies for adding data with document level access control.

* [Using the Add Documents API](#using-the-add-documents-api). Sample scripts are provided which use the Azure AI Search Service Add Documents API to directly manage access control information on _existing documents_ in the index.
* [Using prepdocs and Azure Data Lake Storage Gen 2](#azure-data-lake-storage-gen2-setup). Sample scripts are provided which set up an [Azure Data Lake Storage Gen 2](https://learn.microsoft.com/azure/storage/blobs/data-lake-storage-introduction) account, set the [access control information](https://learn.microsoft.com/azure/storage/blobs/data-lake-storage-access-control) on files and folders stored there, and ingest those documents into the search index with their  access control information.

### Using the Add Documents API

> Manually enable document level access control on a search index and manually set access control values using the [manageacl.ps1](../../scripts/manageacl.ps1) script.

Reviewer: Please use URLs starting with "/" here and in original so we can copy/paste
